src/modules/visTypes.js

```diff
@@ -147,13 +149,26 @@ const twoCategoryChartTypes = [
     VIS_TYPE_STACKED_AREA,
 ]
 
+const columnBasedTypes = [
+    VIS_TYPE_COLUMN,
+    VIS_TYPE_BAR,
+    VIS_TYPE_YEAR_OVER_YEAR_COLUMN,
+    VIS_TYPE_STACKED_COLUMN,
+    VIS_TYPE_STACKED_BAR,
+]
+
 const verticalTypes = [VIS_TYPE_BAR, VIS_TYPE_STACKED_BAR, VIS_TYPE_GAUGE]
 
+const legendSetTypes = [VIS_TYPE_COLUMN, VIS_TYPE_BAR]
```

Comment:
Shouldn't this include PT and SV to be correct?

Reply:
I'm not sure. All "legend" types include PT, SV and Gauge, but what I wanted to do is create a prop that only includes the "regular" types (regular = that has a vertical y axis and horisontal x axis) that support legends, which are only Column and Bar. 
I discussed this with Jan briefly, this could either be renamed (I can't come up with a good name for it though) or used in conjunction with a list of all "regular" types. But I think this really overcomplicates things 🤔 

Open for suggestions to rename this prop though...

Reply:
I see.
I didn't think about Gauge, the other 2 use different engines for generating the visualization.
Keep it simple.